src/renderer/MMM-awesome-alexa.ts

```diff
@@ -25,11 +27,17 @@ Module.register("MMM-awesome-alexa", {
         lite: false,
     },
 
+    visualizer: undefined,
+    canvas: undefined,
```

Comment:
You can remove these initialisations. As you can see from the [docs](https://github.com/MichMich/MagicMirror/tree/master/modules#start)
 Right place to init these variables is in start method
> The start method is a perfect place to define any additional module properties

Reply:
Done :)